demos/netgen/NetGenPoisson.rst

```diff
@@ -0,0 +1,89 @@
+We import the needed library from FireDrake and NetGen.
+
+.. code:: ipython3
+
+    from firedrake import *
+    import firedrake.cython.dmcommon as dmcommon
+    from firedrake import mesh
+    from ngsolve import Mesh,Draw
+    from netgen.geom2d import CSG2d, Circle, Rectangle
```

Comment:
make sure you don't have redundant imports. For example likes 6 and 7 are definitely not needed.